docs/TERMS.md

```diff
@@ -0,0 +1,87 @@
+
+## Indemnification
+
+You agree to indemnify and hold harmless COMP, its affiliates, contractors, and its licensors, and their respective directors, officers, employees and agents from and against any and all claims and expenses, including attorneys’ fees, arising out of your use of this Agreement, including but not limited to your violation of this Agreement.
+
+## Disclaimer of Warranty
+
+The Website is provided “as is”. COMP and its affiliates, suppliers and licensors hereby disclaim all warranties of any kind, express or implied, including, without limitation, the warranties of merchantability, fitness for a particular purpose and non-infringement. Neither COMP nor its suppliers and licensors, makes any warranty that the Website will be error free or that access thereto will be continuous or uninterrupted. You understand that you download from, or otherwise obtain content or services through, the Website at your own discretion and risk.
+
+## Partial Invalidity
+
+If any provision of this document is held by a court of competent jurisdiction to be invalid, void, or unenforceable, the remaining provisions shall nevertheless continue in full force without being impaired or invalidated in any way.
+
+## Failure to Enforce
+
+The failure of either party to enforce at any time, or for any period of time, the provisions hereof shall not be construed to be a waiver of such provisions or of the right of such party to enforce each and every such provision.
+
+## Dispute Resolution
+
+The parties will engage in good faith negotiations to resolve the dispute for a period of ten (10) business days after written notice of the dispute or issue is provided by one party to the other. Within such ten (10) business days, representatives from each party will engage in negotiations to resolve the dispute, and such individuals will meet in person, via videoconference or via telephone and attempt to informally resolve the dispute or issues. If those persons are unable to resolve the dispute within such ten (10) business days, then unless the parties have mutually agreed to extend the negotiation period, a party may exercise its rights available to such party under this Agreement to otherwise end the dispute.
+
+
+## Governing law
+
+Except to the extent applicable law provides otherwise, this Agreement between you and GitHub and any access to or use of the Website or the Service are governed by the federal laws of the United States of America and the laws of the State of New Jersey, without regard to conflict of law provisions. You and COMP agree to submit to the exclusive jurisdiction and venue of the courts located in the City of Atlantic Highlands, NJ.
```

Comment:
Should "you and GitHub" be "you and COMP"?

Reply:
Resolved.